src/TimeModeling/Types/judiVector.jl

```diff
@@ -432,7 +435,7 @@ function judiVector_to_SeisBlock(d::judiVector{avDT, AT}, q::judiVector{avDT, QT
         # set headers
         set_header!(blocks[j], "GroupX", convert(Array{Integer,1},round.(d.geometry.xloc[j]*1f3)))
         if length(d.geometry.yloc[j]) == 1
-            set_header!(blocks[j], "GroupY", Int(round(d.geometry.yloc[j]*1f3)))
+            set_header!(blocks[j], "GroupY", Int.(round.(d.geometry.yloc[j]*1f3)))
```

Comment:
This is the only part relevant to #85, and it's not fixed correctly. Length is 1 so should just be `d.geometry.yloc[j][1]`. Doesn't explain the rest.